config/checkstyle-suppressions.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE suppressions PUBLIC "-//Puppy Crawl//DTD Suppressions 1.1//EN"
```

Comment:
And here still revers to Puppy Crawl as well

Reply:
done